index.html

```diff
@@ -5,66 +5,180 @@
   <meta name="author" content="Elyse Adrienne Hadid" />
   <meta http-equiv="refresh" content="30" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="reset.css" />
+  <link rel="stylesheet" href="index.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Martel+Sans:wght@200;300;400;600;700;800;900&display=swap"
+    rel="stylesheet"
+  />
```

Comment:
We should remove font-weights we aren't using since we'll never look at this again